deploy/cdk/step_function_stack.py

```diff
@@ -165,6 +165,12 @@ def _publication_workflow(
             output_path="$.Payload",
         )
 
+        build_stac_item_task.add_retry(
+            errors=["RasterioIOError"],
+            interval=core.Duration.seconds(2),
+            max_attempts=5
+        )
```

Comment:
Very clean, very simple. I like that we can select conditions for retry w/ exceptions